guides/common/modules/proc_configuring-user-federation-for-project-in-keycloak.adoc

```diff
@@ -0,0 +1,30 @@
+[id="configuring-user-federation-for-project-in-keycloak_{context}"]
+= Configuring User Federation for {Project} in {Keycloak}
+Use this procedure to configure user federation for {Project} in the {Keycloak} web UI.
+
+.Procedure
+
+. Log in to a {Keycloak} administer account.
+. At the top-left of the screen, from the drop-down menu, select the realm for {Project} user accounts.
+. Click *User Federation*.
+. From the *Add Provider* list, select *ldap*.
+. From the *Vendor* list, select *Active Directory*.
+. From the *Edit Mode* list, select *UNSYNCED*.
+. In the *Username LDAP attribute* field, enter _*sAMAccountName*_.
+. In the *RDN LDAP attribute* field, enter _*sAMAccountName*_.
+. In the *UUID LDAP attribute* field, enter the name of LDAP attribute that is used s a unique object identified for objects in LDAP.
+. In the *User Object Classes* field, enter all values of object classes from your LDAP server.
+. In the *Connection URL* field, enter the connection URL of your LDAP server, for example, *ldaps://_ldap.server.example.com_:636*.
+. In the *Users DN* field, enter users DN from your LDAP server.
+. From the *Bind Type* list, select *simple*.
+. In the *Bind DN* field, enter DN of LDAP administer user.
+. In the *Bind Credential* field, enter the password of LDAP administer user.
+. Optional: In the *Custom User LDAP Filter* field, enter the filter.
+. Set *Enable StartTLS* to *OFF*. What's this??
```

Comment:
I cannot find this setting in the RHSSO web UI.

Reply:
Lets leave it out from the documentation for now. Maybe it was in a previous version and now it does not appear.